index.html

```diff
@@ -360,108 +360,73 @@ <h3>
             [Default] object toJSON();
         };
         </pre>
-        <p data-dfn-for='PerformanceNavigationTiming'>
-          When getting the value of the <dfn>unloadEventStart</dfn> attribute,
-          run the following steps:
+
+        <p>A <a>PerformanceNavigationTiming</a> has an associated
+        [=document load timing info=] <a data-dfn-for="PerformanceResourceTiming">
```

Comment:
Broken link

Reply:
I think you need to add HTML to the xref specs: https://github.com/w3c/navigation-timing/pull/141/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L70